javalanglib/src/main/scala/java/lang/StringBuilder.scala

```diff
@@ -100,6 +100,10 @@ class StringBuilder(private var content: String) extends CharSequence
     this
   }
 
+  def ensureCapacity(minimumCapacity: Int): Unit = {
+    // Do nothing
+  }
```

Comment:
Believe it or not, this needs a test :p Basically a test that just calls the method, so that we test that it links.

Also, please add the same method in `StringBuffer` as well, and its test (we keep them in sync).